test/test_app_log.c

```diff
@@ -872,7 +877,7 @@ void test_app_log_config_set_notinit (void)
     err_code |= app_log_config_set (&defaults);
     TEST_ASSERT (RD_ERROR_INVALID_STATE == err_code);
 }
-
+#endif  //  unused functions should not be tested    /\    /\   /\
```

Comment:
Remove

Reply:
I thought we agreed to cancel this previously ?